web_src/js/components/RepoActionView.vue

```diff
@@ -163,7 +163,13 @@ const sfc = {
     },
     // rerun a job
     rerunJob(idx) {
-      this.fetch(`${this.run.link}/jobs/${idx}/rerun`);
+      const jobLink = `${this.run.link}/jobs/${idx}`;
+      this.fetch(`${jobLink}/rerun`, null, () => {
```

Comment:
ps: the name `fetch` seems misleading. When  reading the code, it seems a `GET` fetch request.

`fetchPost` seems more clear for its name.

And I  think you do not need the `callback`, you can just do:

```
await fetchPost(...)
reload(...)
```

Reply:
👍🏻 Done.